pyvista/plotting/widgets.py

```diff
@@ -269,7 +269,7 @@ def callback(planes):
             interaction_event=interaction_event,
         )
 
-        return self.add_mesh(box_clipped_mesh, reset_camera=False, **kwargs)
+        return self.add_mesh(box_clipped_mesh, reset_camera=False, copy_mesh=False, **kwargs)
```

Comment:
You no longer need to change these lines because these are the default

Reply:
Ah yes, I've reverted the change